_posts/2019-12-20-reading.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Reading
+date: 2019-12-20 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+When I was a teen I had a lot of free time after school, way more than I have now working a full time job.
+
+Most kids my age used to go out and drink. I used to stay home and read. I would also play computer games from time to time, but not as often as I would be reading.
+
+In this last few months I'm still trying to read whenever I can, but up to last year I would read one or two books a year. This year I've read 5 books so far and I'm on track to read at least one more by the end of the year. I could read way more if I forced myself to, but I don't want to force it too much.
+
+Reading needs to be something that you enjoy doing and you like doing, not something that you do just because you set a number of books to read in a specific timeframe.
+
+You need to read with an open mind and will. Having a forced limit is counterproductive to that objective.
```

Comment:
Can say what changed? Just the time? No will? Nothing interesting to read?

What do you read? Any recommendations? 

How about refusing to read just because you watched a botched TV version of the book :p

Reply:
>How about refusing to read just because you watched a botched TV version of the book :p

GOT? Think I will still read it when (if) it comes out. Or in general, in general never really happened..?

Reply:
Expanse... the book is WAY better and the show is already perfect so imagine!